internal/db/migrations/000013_add_front_end_to_joy_id_infos.up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE joy_id_info_versions
+    ADD front_end varchar(255) NOT NULL AFTER alg;
```

Comment:
`old_front_end` is needed for joy_id_info_versions to restore `front_end` when CKB forks

Reply:
my mistake, it is done now